concepts/40 Angular Components/10 Getting Started/03 Add DevExtreme to an Angular CLI Application/03 Configure Stylesheets.md

```diff
@@ -24,6 +24,18 @@ Open the `angular.json` file and reference `dx.common.css` and a [predefined the
       ...
     }
 
+Then, go to the `src` folder, open the `index.html` file, and add the `dx-viewport` class to the `<body>` tag. This ensures that theme colors and typography settings are applied to all elements on the page, not only to DevExtreme widgets.
+
+    <!-- tab: index.html -->
+    <html lang="en">
+        <head>
+            <!-- ... -->
+        </head>
+        <body class="dx-viewport">
+            <app-root></app-root>
+        </body>
+    </html>
+
 **For Angular CLI before 6**, modify the `angular-cli.json` file instead:
```

Comment:
By "For Angular CLI before 6" do you mean "For Angular CLI (prior to Angular 6)"

Reply:
We are dropping support for Angular versions lower than 7, so this part will be removed.

Reply:
Okay